src/components/DocumentationTopic/Description.vue

```diff
@@ -10,22 +10,13 @@
 
 <template>
```

Comment:
Because Description.vue no longer has a special meaning, no special Vue logic, apart from a few CSS lines, I would suggest we refactor it into a plain `div.description`. We can even go as far as hide it if its empty with css
```css
.description:empty { display: none }
```

Reply:
Is hiding it necessary if it is empty and already has no margin/padding?